server/src/main.rs

```diff
@@ -127,6 +127,14 @@ impl LanguageServer for Backend {
             .log_message(MessageType::INFO, "Slice Language Server config changed")
             .await;
 
+        // Check if disableLanguageServer is set to true
+        if !self.should_enable_language_server(params.clone()).await {
```

Comment:
Can this case actually happen?

Reply:
I think your right, not seeinng the logs on the server so its shutdown before this even fires.